build-system/tasks/extension-generator/extension-doc.template.md

```diff
@@ -0,0 +1,220 @@
+---
+\$category@: presentation
+formats:
+  - websites
+  - email
```

Comment:
Remove

Reply:
The template is a demonstration of format filtering, it isn't Bento specific. It should probably include all formats...

Reply:
Yeah that's a good point